### PR TITLE
scripts/mc_megaraid_monitor.py: script to monitor RAID statuses

### PR DESCRIPTION
**Pull request by pkgw**
Part of M&C's job is to check on the status of our RAID arrays. This script uses `storcli64` to assess the RAID status and log it to the database. There is all sorts of information that we *could* log, but hopefully this analysis covers the most important things by watching the event log for errors.

### CODE REVIEW

File: scripts/mc_megaraid_monitor.py [hunk 1]
```diff
@@ -0,0 +1,244 @@
...
+
+
+# Before running anything, make sure we can connect to the DB.
+
+parser = mc.get_mc_argument_parser()
+args = parser.parse_args()
+db = mc.connect_to_mc_db(args)
+
+
+# Get the seqNum of the last even that we've noticed
```
Comment (bhazelton) [inline scripts/mc_megaraid_monitor.py]: even -> event

File: scripts/mc_megaraid_monitor.py [hunk 2]
```diff
@@ -0,0 +1,244 @@
...
+    #----------------------------------------------------------------
+    #0/0   RAID60 Optl  RW     Yes     RWBD  -   ON  196.475 TB
+    #----------------------------------------------------------------
+
+    if line.startswith('0/0'):
+        item_values['VD 0/0 State'] = line.split()[2]
+
+if show_all.wait() != 0:
+    print('error: storcli exited with an error code', file=sys.stderr)
+    print('unfortunately this script may have swallwed its error message', file=sys.stderr)
```
Comment (bhazelton) [inline scripts/mc_megaraid_monitor.py]: swallwed -> swallowed

File: scripts/mc_megaraid_monitor.py [hunk 3]
```diff
@@ -0,0 +1,244 @@
...
+    TBD: I think these times are in system local time (and not, say, UTC), but
+    it's not clear!
+
+    """
+    dow, month, dom, hhmmss, year = text.split()
+    month = _months[month]
+    dom = int(dom)
+    year = int(year)
+    iso_format = '%04d-%02d-%02d %s' % (year, month, dom, hhmmss)
+    return Time(iso_format, format='iso')
```
Comment (bhazelton) [inline scripts/mc_megaraid_monitor.py]: What time zone gets attached when you call this? The system local time? Or do you need to specify it somehow?

Comment (pkgw) [inline scripts/mc_megaraid_monitor.py]: Good call, needed to be a bit more careful here. Addressed in a80bbcc.

File: scripts/mc_megaraid_monitor.py (hunk 1 above)
Comment (pkgw) [inline scripts/mc_megaraid_monitor.py]: Addressed in 67f2584.

File: scripts/mc_megaraid_monitor.py (hunk 2 above)
Comment (pkgw) [inline scripts/mc_megaraid_monitor.py]: Addressed in 67f2584.

**Pull request merged**